MetadataExtractor/Formats/QuickTime/QuickTimeReaderExtensions.cs

```diff
@@ -58,5 +58,21 @@ public static decimal Get32BitFixedPoint([NotNull] this SequentialReader reader)
                 reader.GetUInt16(),
                 decimal.Divide(reader.GetUInt16(), ushort.MaxValue));
         }
+
+        public static int[] GetMatrix(this SequentialReader reader)
+        {
+            var matrix = new int[9];
+            for (var i = 0; i < matrix.Length; i++)
+            {
+                var val = reader.GetInt32() / ushort.MaxValue;
+                // the right column is fixed 2.30 instead of 16.16
```

Comment:
Do you have a reference for this info?

Reply:
Added comment with link to the specification.

Reply:
Perfect, thank you.